web/apps/apps.go

```diff
@@ -107,10 +107,12 @@ func getHandler(appType consts.AppType) echo.HandlerFunc {
 		if err := middlewares.Allow(c, permission.GET, man); err != nil {
 			return err
 		}
+
+		registries := instance.Registries()
+		copier := app.Copier(consts.WebappType, instance)
+		man = app.DoLazyUpdate(instance, man, copier, registries)
+
 		if appType == consts.WebappType {
-			registries := instance.Registries()
-			copier := app.Copier(consts.WebappType, instance)
-			man = app.DoLazyUpdate(instance, man, copier, registries)
 			man.(*app.WebappManifest).Instance = instance
```

Comment:
I believe this can be removed as this is taken care of by `WebappManifest.ReadManifest()` which is called during the webapp `install()` and `update()` steps.

See https://github.com/cozy/cozy-stack/blob/master/model/app/webapp.go#L301

Reply:
I can't be sure this will not break something so at the moment I would prefere keeping this line.